GatherBuddy.GameData/Classes/Fish.CatchData.cs

```diff
@@ -7,24 +7,27 @@ namespace GatherBuddy.Classes;
-    public Fish[]            Mooches         { get; internal set; } = [];
-    public (Fish, int)[]     Predators       { get; internal set; } = [];
-    public int               IntuitionLength { get; internal set; } = 0;
-    public RepeatingInterval Interval        { get; internal set; } = RepeatingInterval.Always;
-    public Snagging          Snagging        { get; internal set; } = Snagging.Unknown;
-    public Lure              Lure            { get; internal set; } = Lure.None;
-    public HookSet           HookSet         { get; internal set; } = HookSet.Unknown;
-    public BiteType          BiteType        { get; internal set; } = BiteType.Unknown;
-    public SpearfishSize     Size            { get; internal set; } = SpearfishSize.Unknown;
-    public SpearfishSpeed    Speed           { get; internal set; } = SpearfishSpeed.Unknown;
-    public Fish?             SurfaceSlap     { get; internal set; }
-    public string            Guide           { get; internal set; } = string.Empty;
-    public OceanTime         OceanTime       { get; internal set; } = OceanTime.Always;
-    public int               MultiHook       { get; internal set; }
+    public Patch             Patch              { get; internal set; } = Patch.Unknown;
+    public Weather[]         PreviousWeather    { get; internal set; } = [];
+    public Weather[]         CurrentWeather     { get; internal set; } = [];
+    public Bait              InitialBait        { get; internal set; } = Bait.Unknown;
+    public Fish[]            Mooches            { get; internal set; } = [];
+    public (Fish, int)[]     Predators          { get; internal set; } = [];
+    public int               IntuitionLength    { get; internal set; } = 0;
+    public RepeatingInterval Interval           { get; internal set; } = RepeatingInterval.Always;
+    public Snagging          Snagging           { get; internal set; } = Snagging.Unknown;
+    public Lure              Lure               { get; internal set; } = Lure.None;
+    public HookSet           HookSet            { get; internal set; } = HookSet.Unknown;
+    public BiteType          BiteType           { get; internal set; } = BiteType.Unknown;
+    public SpearfishSize     Size               { get; internal set; } = SpearfishSize.Unknown;
+    public SpearfishSpeed    Speed              { get; internal set; } = SpearfishSpeed.Unknown;
+    public Fish?             SurfaceSlap        { get; internal set; }
+    public string            Guide              { get; internal set; } = string.Empty;
+    public OceanTime         OceanTime          { get; internal set; } = OceanTime.Always;
+    public int               MultiHook          { get; internal set; }
+    public int               MultiHookExtension { get; internal set; }
+    public int               Points             { get; internal set; }
 
     internal OptionalBool BigFishOverride { get; set; } = null;
+    internal OptionalBool LegendaryFish   { get; set; } = null;

```

Comment:
No reason to have that as optional, just make it a bool. If any optional defaults to false anyway and we don't take another value for this there's no difference.